### ISSUE TITLE
bundle options under options.a11y

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/a11y-research#100]
From https://github.com/phetsims/area-model-common/issues/139#issuecomment-387182176...

Instead of relying on developers to manually group a11y-related options, consider bundling (nesting) them under `options.a11y`, e.g.:

```js
options = _.extends( {

  fill: ‘white’,
  stroke: ‘black’,
  …

  phetio: {
    tandem: tandem
    ...
  }
  …

  a11y: { 
    tagName: 'p',
    descriptionContent: 'blah blah blah',
    …
  }

}, options );
```

**Comment by samreid** [comment]
I recommend we conclude discussion in https://github.com/phetsims/area-model-common/issues/139 before proceeding here.

**Comment by zepumph** [comment]
We have decided to go with code comments instead of nested options. CLosing
